src/libs/Network/index.js

```diff
@@ -181,6 +182,7 @@ function post(command, data = {}, type = CONST.NETWORK.METHOD.POST, shouldUseSec
             ...data,
             shouldRetry: lodashGet(data, 'shouldRetry', true),
             canCancel: lodashGet(data, 'canCancel', true),
+            appversion: version,
```

Comment:
Should this be `camelCase`?

```suggestion
            appVersion: version,
```

Reply:
Nope 😄 we specifically check for all lowercase in the API https://github.com/Expensify/Web-Expensify/blob/9c2315965d377146fe576345609fd817368d4cd1/lib/Request.php#L86